### DISCUSSION TITLE
How to solve this problem?

### DISCUSSION THREAD

**Discussion by xiaozhen228** [obss/sahi#858] [Q&A] [answered]
shapely.errors.GEOSException: TopologyException: Input geom 0 is invalid: Self-intersection at 1191.7142857142858 1183.4285714285713

**Comment by austinmw** [comment]
Bump, it would be nice to catch then skip on this exception

**Comment by ccalvocm** [comment, marked as answer]
Hello @xiaozhen228: 

Does Shapely's geometry.buffer(0) solves your topology issue?.

Let me know.
